graphql/getter.js

```diff
@@ -60,6 +62,27 @@ export default class Getter {
     return this;
   };
 
+  withBm25 = (bm25Obj) => {
+    try {
+      this.bm25String = new Bm25(bm25Obj).toString();
```

Comment:
`this.bm25String` is never used, therefore it is not present in final request sent to server